RecoEcal/EgammaCoreTools/interface/ClusterEtLess.h

```diff
@@ -1,18 +1,12 @@
 #ifndef RecoECAL_ECALClusters_ClusterEtLess_h
 #define RecoECAL_ECALClusters_ClusterEtLess_h
 
-
-#include "DataFormats/CaloRecHit/interface/CaloCluster.h"
+namespace reco {
+    class CaloCluster;
+}
 
 // Less than operator for sorting EcalRecHits according to energy.
-class ClusterEtLess : public std::binary_function<reco::CaloCluster, reco::CaloCluster, bool>
-{
- public:
-  bool operator()(reco::CaloCluster x, reco::CaloCluster y)
-    {
-      return ( (x.energy() * sin(x.position().theta())) < (y.energy() * sin(y.position().theta())) ) ;
-    }
-};
+bool isClusterEtLess(reco::CaloCluster x, reco::CaloCluster y);
```

Comment:
was the copy elided here? (else, it's unclear why we pass x and y by value)

Reply:
I copied it from the previous implementation in the header file. Will change.